### PR TITLE
Updates to latest CF

### PR DESCRIPTION
**Pull request by anselmbradford**
Updates to latest CF to pull in large target form inputs.

Pull this branch and run `grunt && npm start`

download

### CODE REVIEW

**Review by cfarm — APPROVED** [review]

File: src/static/js/app.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
    ========================================================================== */
 
 var $ = require( 'jquery' );
-require( 'cf-expandables' );
-require( 'cf-tables' );
+require( 'cf-expandables/src/Expandable' );
+require( 'cf-tables/src/Table' );
```
Comment (virginiacc) [inline src/static/js/app.js:8]: If this supercedes #615, should it pull in the code from that PR to initialize the expandables and tables? 

```
var Expandable = require( 'cf-expandables/src/Expandable' );
var Table = require( 'cf-tables/src/Table' );

Expandable.init();
Table.init();
```

Comment (cfarm) [reply src/static/js/app.js:8]: good catch!

**Review by cfarm — CHANGES_REQUESTED** [review]
see Virginia's comment for fix needed from #615

File: src/static/js/app.js (hunk 1 above)
Comment (anselmbradford) [reply src/static/js/app.js:8]: I'll merge this and then go over to 615 and resolve conflicts.

Comment (anselmbradford) [reply src/static/js/app.js:8]: @cfarm you okay with this?

Comment (cfarm) [reply src/static/js/app.js:8]: sure

**Review by cfarm — APPROVED** [review]

**Pull request merged**